pom.xml

```diff
@@ -151,6 +151,11 @@
                 <artifactId>prettytime</artifactId>
                 <version>3.2.5.Final</version>
             </dependency>
+            <dependency>
+                <groupId>org.kohsuke.metainf-services</groupId>
+                <artifactId>metainf-services</artifactId>
+                <version>1.1</version>
```

Comment:
I'm OK with this version. I see that the last version is 1.7 (http://mvnrepository.com/artifact/org.kohsuke.metainf-services/metainf-services)